gr-blocks/lib/CMakeLists.txt

```diff
@@ -253,6 +253,9 @@ if(ENABLE_TESTING)
     GR_ADD_CPP_TEST("blocks_${qa_file}"
       ${CMAKE_CURRENT_SOURCE_DIR}/${qa_file}
     )
+    set_tests_properties("blocks_${qa_file}"
+        PROPERTIES TIMEOUT 30
+    )
```

Comment:
I don't think this change needs to be a part of this PR.

Reply:
This is required for the test to fail.
There is no default timeout, so the test would hang indefinitely.

Reply:
OK, if there's no timeout at all, I suppose it's a good improvement to the test suite.